firmware/application/apps/ui_freqman.cpp

```diff
@@ -83,24 +77,21 @@ void FreqManBaseView::get_freqman_files() {
 };
 
 void FreqManBaseView::change_category(int32_t category_id) {
-    if (!file_list.size()) return;
-
     current_category_id = category_id;
 
+    if (!file_list.size()) return;
+
     std::vector<freqman_entry>().swap(database);
 
-    if (!load_freqman_file(file_list[categories[current_category_id].second], database)) {
+    if (!load_freqman_file(file_list[categories[category_id].second], database)) {
         error_ = ERROR_ACCESS;
     }
     menu_view.set_db(database);
+    if (database.size() == 0)
```

Comment:
nit: `text_empty.hidden(database.size());`

Reply:
why not

Reply:
Ended up with this:
 text_empty.hidden(!database.empty());